Added some kitchen clutter

Drafting-Goblin opened the pull request:
#### Summary

Summary: Content "Added pizza cutters, icecream scoops, and cheese graters. they will spawn in some domestic drawers."

#### Purpose of change

The objects that spawned in kitchens felt consistently too similar. This was also in the "List of easy jobs". 

#### Describe the solution

This will cause pizza cutters, icecream scoops and cheese graters to spawn in some domestic drawers. These are items that some people own, so they have a lower spawn rate than other items. Sponges will now spawn under kitchen sinks.

#### Describe alternatives you've considered

I thought of a few items to add, but these seemed fairly simple and they felt like a good starting place. 

#### Testing

I compiled my changes and I made sure the items both spawned naturally in the game and could be spawned via the debug menu. Tested the deconstruct recipies.

I-am-Erk commented on data/json/recipes/recipe_deconstruction.json:
```diff
@@ -5021,5 +5021,14 @@
+    "skill_used": "fabrication",
+    "time": "12 s",
+    "qualities": [ { "id": "SCREW", "level": 1 } ],
+    "components": [ [ [ "spring", 2 ] ] ]
```
```suggestion
    "components": [ [ [ "spoon", 1 ] ], [ [ "spring", 2 ] ] ]
```

Qrox commented on data/json/items/generic/dining_kitchen.json:
```diff
@@ -1061,5 +1061,26 @@
+    "type": "GENERIC",
+    "category": "other",
+    "id": "pizza_cutter",
+    "name": { "str": "pizza cutter", "str_pl": "pizza cutters" },
```
```suggestion
    "name": { "str": "pizza cutter" },
```

Qrox commented on data/json/items/generic/dining_kitchen.json:
```diff
@@ -1061,5 +1061,26 @@
+    "category": "other",
+    "id": "pizza_cutter",
+    "name": { "str": "pizza cutter", "str_pl": "pizza cutters" },
+    "description": "A utensil used for cutting pizza. practically useless in combat.",
```
```suggestion
    "description": "A utensil used for cutting pizza.  Practically useless in combat.",
```

Qrox commented on data/json/items/generic/dining_kitchen.json:
```diff
@@ -1061,5 +1061,26 @@
+    "type": "GENERIC",
+    "category": "other",
+    "id": "icecream_scoop",
+    "name": { "str": "icecream scoop", "str_pl": "icecream scoop" },
```
```suggestion
    "name": { "str": "icecream scoop" },
```

Qrox commented on data/json/items/generic/dining_kitchen.json:
```diff
@@ -1061,5 +1061,26 @@
+    "category": "other",
+    "id": "icecream_scoop",
+    "name": { "str": "icecream scoop", "str_pl": "icecream scoop" },
+    "description": "A utensil used for scoop icecream into bowls. It has a spring powered lever to assist.",
```
```suggestion
    "description": "A utensil used for scooping icecream into bowls.  It has a spring powered lever to assist.",
```

Qrox commented on data/json/recipes/recipe_deconstruction.json:
```diff
@@ -5021,5 +5021,14 @@
+    "skill_used": "fabrication",
+    "time": "12 s",
+    "qualities": [ { "id": "SCREW", "level": 1 } ],
+    "components": [ [ [ "spoon", 1 ] ], [ [ "spring", 2 ] ] ]
```
`"spring"` is supposed to be a 3 kg heavy-duty spring, so you probably won't get it from an icecream scoop.

Drafting-Goblin replied:
wow, that's an oversight on my part. That's a good point and it would make sense to remove that then. Thank you.

Drafting-Goblin replied:
Springs have been removed from the deconstruct recipe.

I-am-Erk commented:
This needs linting

mlangsdorf reviewed (changes requested):
Please fill out the PR submission header correctly.  It has easy-to-read instructions for your convenience.

Drafting-Goblin commented:
Continuous-integration/travis-ci/pr has failed. It says the job has errored at the Xcode 10.2 Make build with Tiles and sound (macOS). From what I can tell, the errors are: ../tests/event_test.cpp:60: FAILED: REQUIRE( sub.events.size() == 1 ), and, ../tests/event_test.cpp:76: FAILED: CHECK( sub.events.size() == 1 ). I'm unsure how to proceed regarding this. I've tried Googling but I didn't see anything that helped. Any sort of direction or guidance would be greatly appreciated. Thanks.

Qrox commented:
I think that error is not related to this PR. @jbytheway can you look into this? Seems like something's wrong with the event system on Xcode.

jbytheway commented:
That OSX bug is super mysterious. It was a problem for a while.  Kevin spent several days trying to figure out what was wrong to no avail, and eventually fixed it by upgrading the xcode version on Travis.  A couple of weeks ago it came back and we don't know why. I have no idea what to do about it.

jbytheway commented:
See #43043.

Drafting-Goblin commented:
Added a small update to this PR. Added a cheese grater that spawn in domestic kitchen drawers. They can be deconstructed for a plastic chunk and scrap. Also I noticed that sponges didn't spawn under the sink and that seems like a place they would spawn, so I made them spawn there.
